.circleci/config.yml

```diff
@@ -268,40 +268,40 @@ workflows:
       - win_unit_tests:
           matrix:
             parameters:
-              python_version: ["3.8", "3.7", "3.6"]
+              python_version: ["3.6", "3.7", "3.8", "3.9.0rc2"]
```

Comment:
Waiting on CircleCI to release Python 3.9 image
- https://circleci.com/docs/2.0/circleci-images/#python